certora/specs/ERC20Votes.spec

```diff
@@ -48,53 +48,65 @@ hook Sstore _checkpoints[KEY address account][INDEX uint32 index].votes uint224
         lastIndex@new(account) == index;
 }
 
+
 ghost lastFromBlock(address) returns uint32;
 
 ghost doubleFromBlock(address) returns bool {
     init_state axiom forall address a. doubleFromBlock(a) == false;
 }
 
+
+
+
 hook Sstore _checkpoints[KEY address account][INDEX uint32 index].fromBlock uint32 newBlock (uint32 oldBlock) STORAGE {
     havoc lastFromBlock assuming
         lastFromBlock@new(account) == newBlock;
-
-    havoc doubleFromBlock assuming
+    
+    havoc doubleFromBlock assuming 
         doubleFromBlock@new(account) == (newBlock == lastFromBlock(account));
 }
 
-///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-//                                                    Invariants                                                     //
-///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
-
 // sum of user balances is >= total amount of delegated votes
 // fails on burn. This is because burn does not remove votes from the users
```

Comment:
Not sure I understand this comment. Burn should remove votes is the account we are burning from has delegated.